docs/index.rst

```diff
@@ -170,11 +169,10 @@ maintains an up-to-date list.
 .. _Levanter: https://github.com/stanford-crfm/levanter
 .. _Lineax: https://github.com/patrick-kidger/lineax
 .. _MaxText: https://github.com/google/maxtext/
-.. _NNX: https://flax.readthedocs.io/en/latest/nnx/
 .. _Numpyro: https://num.pyro.ai/en/latest/index.html
 .. _Optax: https://optax.readthedocs.io/
 .. _Optimistix: https://github.com/patrick-kidger/optimistix
 .. _Orbax: https://orbax.readthedocs.io/
 .. _PyMC: https://www.pymc.io/
-.. _Tensorflow datasets: https://www.tensorflow.org/datasets
-.. _Tensorflow probabilty: https://www.tensorflow.org/probability
+.. _TensorFlow Datasets: https://www.tensorflow.org/datasets
+.. _TensorFlow Probabilty: https://www.tensorflow.org/probability
```

Comment:
If your intent is to change how these appear on the page, you'll also have to change the places where they are referenced in the text above.

Reply:
On it.

Reply:
Done @jakevdp